org.uqbar.project.wollok.tests/src/org/uqbar/project/wollok/tests/sdk/ObjectTest.xtend

```diff
@@ -31,11 +31,10 @@ class ObjectTest extends AbstractWollokInterpreterTestCase {
 				
 				const anonymousObject = object {
 					var edad = 23
-					var altura = 1.7
+					var altura = 2
 				}
 				
-«««				//TODO
-«««				assert.equals("anObject[edad=23, altura=1.7]", anonymousObject.toString())
+				assert.equals("an Object[edad=23, altura=2]", anonymousObject.toString())
```

Comment:
Mmm I don't know if I understood. But I don't want an special toString() method for objects with Double fields :P.

Anyway, I was thinking do it in the PR of https://github.com/uqbar-project/wollok/issues/771